### DISCUSSION TITLE
Tidak Bisa Scan QR Code

### DISCUSSION THREAD

**Discussion by nana7272** [kirimwa-id/forum#63] [Tanya Jawab] [answered]
Saya tidak bisa connect saat scan QRCode, selalu muncuk tulisan "Please Scan The Code Again", Padahal kemarin sempat bisa konek dan bertahan 1hari, hari ini disconnect sya liat satus devicenya trs reconect dan scan qrcode lgi tidak bisa

[https://s3-id-jkt-1.kilatstorage.id/umum/WhatsApp%20Image%202021-11-02%20at%2013.32.59.jpeg](url)
[https://s3-id-jkt-1.kilatstorage.id/umum/WhatsApp%20Image%202021-11-02%20at%2013.32.59%20%281%29.jpeg](url)
[https://s3-id-jkt-1.kilatstorage.id/umum/WhatsApp%20Image%202021-11-02%20at%2013.32.59%20%282%29.jpeg](url)

**Comment by fahram** [comment]
Ini pakai whatsapp yang Join Multi Device (beta) bukan ?

**Comment by nana7272** [comment]
Bukan, WA Join Multi Device (beta) tidak saya aktifkan, dan kemarin sempat konek 1 hari tidak ada masalah, mangkanya kok aneh gk bisa scan QRCode, sudah sya coba di 3 Hp tetap tidak bisa.

**Reply by fahram** [reply]
Oh iya bener. Barusan saya coba memang tidak bisa

**Comment by nana7272** [comment]
[https://s3-id-jkt-1.kilatstorage.id/umum/WhatsApp%20Image%202021-11-02%20at%2013.56.58.jpeg](url)
[https://s3-id-jkt-1.kilatstorage.id/umum/WhatsApp%20Image%202021-11-02%20at%2013.56.59.jpeg](url)

**Comment by rioastamal** [comment, marked as answer]
@nana7272 terima kasih laporannya. Masalah telah diperbaiki, silahkan dicoba lagi.

**Reply by nana7272** [reply]
Baik terimakasih @rioastamal
